Modules/Invoices/Controllers/CronController.php

```
@@ -32,15 +26,11 @@ class CronController
      * @param UploadService $uploadService
      */
     public function __construct(
-        InvoiceService $invoiceService,
-        InvoicesRecurringService $invoicesRecurringService,
-        EmailTemplateService $emailTemplateService,
-        UploadService $uploadService
+        protected InvoiceService $invoiceService,
+        protected InvoicesRecurringService $invoicesRecurringService,
```

PHPMD (2.15.0)

Avoid excessively long variable names like $invoicesRecurringService. Keep variable name length under 20. (undefined)

(LongVariable)